extensions/wikia/TemplateDraft/controllers/TemplateDraftController.class.php

```diff
@@ -35,4 +35,23 @@ public function createDraftContent( Title $title, $content, Array $flags ) {
 
 		return $newContent;
 	}
+
+	public function markTemplateAsNotInfobox() {
```

Comment:
And it should also meet the same permission requirements as the display of the module, so should also check for power user.